5.0/fr/0x10-V1-Encoding-and-Sanitization.md

```diff
@@ -0,0 +1,103 @@
+# V1 Encodage et nettoyage
+
+## Objectif du contrôle
+
+Ce chapitre aborde les failles de sécurité les plus courantes des applications web liées au traitement non sécurisé de données non fiables. Ces failles peuvent entraîner diverses vulnérabilités techniques, où les données non fiables sont interprétées selon les règles syntaxiques de l'interpréteur concerné.
+
+Pour les applications web modernes, il est toujours préférable d'utiliser des API plus sûres, telles que les requêtes paramétrées, l'échappement automatique ou les Framework de création de modèles. Dans le cas contraire, un encodage, un échappement ou une purification de sortie soigneusement effectués deviennent essentiels à la sécurité de l'application.
```

Comment:
"purification" : purification is a tough word. "Nettoyage" is probably fine (as in the title).